PyInstaller/utils/conftest.py

```diff
@@ -516,7 +507,10 @@ def pyi_modgraph():
 @pytest.fixture(params=['onedir', 'onefile'])
 def pyi_builder(tmpdir, monkeypatch, request, pyi_modgraph):
     # Save/restore environment variable PATH.
-    monkeypatch.setenv('PATH', os.environ['PATH'], )
+    monkeypatch.setenv(
+        'PATH',
+        os.environ['PATH'],
+    )
```

Comment:
Would have `yapf` decided differently here had original line not contain redundant trailing comma?